pages/deduplicate/views/index.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Fragment } from 'react';
+import { connect } from 'react-redux';
+import classnames from 'classnames';
+import get from 'lodash/get';
+import { Header, Search, Snippet, Link } from '@asl/components';
+
+const dobsDiffer = (profile1, profile2) => profile1.dob && profile2.dob && profile1.dob !== profile2.dob;
+const bothActivePils = (profile1, profile2) => get(profile1, 'pil.status') === 'active' && get(profile2, 'pil.status') === 'active';
+
+class Index extends React.Component {
+
+  onSubmit(p, e) {
+    e.preventDefault();
+    if (bothActivePils(p, this.props.profile)) {
+      return window.alert('These profiles cannot be merged as both have active PILs');
+    }
+    if (dobsDiffer(p, this.props.profile)) {
+      if (window.confirm('These profiles have different dates of birth, are you sure you want to continue?')) {
+        return e.target.submit();
+      }
+    }
+    if (window.confirm('This action cannot be undone')) {
+      e.target.submit();
+    }
+  }
+
+  render () {
+    const { profile, action, alternates, search } = this.props;
+
+    return <Fragment>
+      <Header subtitle={`${profile.firstName} ${profile.lastName}`} title="Merge profile" />
+      <p>Merging this profile will transfer all establishment affiliations, licences and roles to another profile.</p>
```

Comment:
So the profile you click the "dedupe" button on gets merged into the profile you search for, and then gets deleted?

Reply:
Ja

Reply:
Nein.

Es ist nicht deleten.